models/sessions/scratch/snowplow_unified_sessions_this_run.sql

```diff
@@ -471,7 +472,16 @@ on f.session_identifier = a.session_identifier
 {%- if var('snowplow__conversion_events', none) %}
 left join session_convs d on f.session_identifier = d.session_identifier
 {%- endif %}
-
+{% if var('snowplow__enable_mobile_context') %}
```

Comment:
I'd recommend adding {%- 

this is to be consistend with the function above on line 272 

this is for the the code is compiled into sql it will be easier to read